### PR TITLE
Manual deployment warnings

### PR DESCRIPTION
**Pull request by Zaxounette**
Relates to #769 .
Adresses the initial issue, but not the suggestion in https://github.com/btcpayserver/btcpayserver-doc/issues/769#issuecomment-727656623 

* Convert an existing sentence into a warning in `Deployment.md`
* Add a short bold indication in the options bullet-points in `Deployment.md`
* Copy warning from `ExtendedManualDeployment.md` into `ManualDeployment.md`, just under the first H1
* Include an additional sentence to be explicit the community will not provide extensive support/support time for this deployment
* Add this additional sentence to already existing warning in `ManualDeploymentExtended.md`
* Add warning in `Troubleshooting.md` in the "Ask for help" section.

### CODE REVIEW

**Comment by Zaxounette** [comment]
`├─BROKEN─ https://docs.drupalcommerce.org/commerce2/developer-guide/install-update/installation (BLC_UNKNOWN)` is unrelated to this PR.
Invalid certificat, the link works fine if certificat warning is discarded.

File: docs/Troubleshooting.md [hunk 1]
```diff
@@ -129,6 +129,10 @@ Here's a good example of how to ask a question.
 
 > I'm having a problem with XYZ. I can replicate the problem. My BTCPay version is 0.100.31, and I deployed my server on Digital Ocean by following Docker deployment guide. I've searched through the FAQ and closed GitHub issues, but there's no solution to my problem. My BTCPay Setup is XYZ, and the issue is occurring when I do XYZ. Here are the logs I was able to get from my BTCPay instance. You can see the error in the image I attached.
 
+:::danger
+The community will only provide extensive support for production deployments. i.e. [Manual Deployments](ManualDeployment.md) are not intended for production use.
```
Comment (britttttk) [inline docs/Troubleshooting.md:133]:
```suggestion
The community will not provide extensive support for custom deployments. i.e. Variations of [Manual Deployments](ManualDeployment.md) are expected to be used only for development purposes and by users with technical literacy with the ability to **resolve deployment and maintenance issues on their own**. 
```

**Review by britttttk — CHANGES_REQUESTED** [review]
@Zaxounette awesome thanks for finding these warning spots. 

I know this warning is a bit repetitive but this document is linked in the btcpayserver repo's Github new issue page, so I think it would be good to make it very clear here. 

I inverted your statement because technically the community is never required to provide anyone extensive support. It's just FOSS :D but solving all custom issues is definitely not a priority.

File: docs/Troubleshooting.md (hunk 1 above)
Comment (Zaxounette) [reply docs/Troubleshooting.md]: Nice wording :)

**Comment by Zaxounette** [comment]
Failed linkcheck `├─BROKEN─ https://bitcoinshirt.co/btcpay-stores/ (BLC_UNKNOWN)` unrelated to this PR.

**Pull request merged**